bitnami/jenkins/templates/deployment.yaml

```diff
@@ -31,12 +31,18 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
         env:
         - name: JENKINS_USERNAME
-          value: {{ default "" .Values.jenkinsUser | quote }}
+          value: {{ .Values.jenkinsUser | quote }}
         - name: JENKINS_PASSWORD
           valueFrom:
             secretKeyRef:
               name: {{ template "fullname" . }}
               key: jenkins-password
+        - name: JENKINS_HOME
+          value: {{ .Values.jenkinsHome | quote }}
+        - name: DISABLE_JENKINS_INITIALIZATION
+          value: {{ .Values.disableInitialization | quote }}
+        - name: JAVA_OPTS
```

Comment:
Could you check whether `.Values.javaOpts ` is defined and only set *JAVA_OPTS* when it is?